documentation/tut/toy-example-from-scratch.rst

```diff
@@ -178,11 +178,12 @@ Yes, that is quite a large battery :)
 
 .. note:: Obviously, you can use the ``flexmeasures`` command to create your own, custom account and assets. See :ref:`cli`. And to create, edit or read asset data via the API, see :ref:`v3_0`.
 
-We can also look at the battery asset in the UI of FlexMeasures (in Docker, the FlexMeasures web server already runs, on your PC you can start it with ``flexmeasures run``).
-Visit `http://localhost:5000/assets <http://localhost:5000/assets>`_ (username is "toy-user@flexmeasures.io", password is "toy-password") and select "toy-battery":
+We can also look at the dashboard in the UI of FlexMeasures (in Docker, the FlexMeasures web server already runs, on your PC you can start it with ``flexmeasures run``).
```

Comment:
This PR is almost perfect, but one thing bothers me: We changed the picture from now on (v0.14 and up) to show the dashboard in this part of the tutorial.

Older versions will still have the old text, but now point to this picture, which is not what they would see at http://localhost:5000/assets. So older versions of the documentation will be somewhat confusing. 

I think the currently best solution is to add that dashboard picture as a new one called `tut/toy-schedule/asset-view-dashboard.png`, and leave the old asset-view.png intact. So this PR would use the new name, and the sister PR in flexmeasures/screenshots keeps the old image around.

I still believe there might be a more elegant architecture to deal with this documentation versioning problem, but I can't put my finger on it. [Nextcloud had an interesting approach](https://github.com/nextcloud/screenshots/tree/master#readme), but I believe they don't use this repo anymore, really, at least they did not seem to have added a lot of release branches.

Reply:
hmm okay, I will look into that repo.